LICENSE

```diff
@@ -27,6 +27,7 @@ The following files and directories are exceptions:
 * lib/tasks/convert.rake is under Revised BSD license
 * extras/noVNC/websockify is LGPL v3 licensed
 * vendor/assets/fonts/ is ASL 2.0 licensed
+* vendor/assets/fonts/font-awesome is SIL OFL 1.1 licensed
```

Comment:
No need to include this, patternfly.css, nor the LICENSE text if we don't vendor them :smile: